README.md

```diff
@@ -4,6 +4,10 @@ which are a native replacement for annotations. As such, this library is
 considered feature complete, and should receive exclusively bugfixes and
 security fixes.
 
+We do not recommend using this library in new projects and encourage authors
```

Comment:
Let's add `# Project deprecated` or something like this at the top so there's something big screaming "don't use me" :) Also maybe a link to our blogpost https://www.doctrine-project.org/2022/11/04/annotations-to-attributes.html like "This is how Doctrine handled migration" to help folks discover this?

Reply:
I think, the ⚠️ is enough in that regard. I've added a link to the blog.